Source/Core/VideoBackends/D3D/D3DTexture.cpp

```diff
@@ -13,27 +15,31 @@ namespace DX11
 namespace D3D
 {
 
-void ReplaceRGBATexture2D(ID3D11Texture2D* pTexture, const u8* buffer, unsigned int width, unsigned int height, unsigned int pitch, unsigned int level, D3D11_USAGE usage)
+void ReplaceRGBATexture2D(ID3D11Texture2D* pTexture, const u8* buffer, unsigned int width, unsigned int height, unsigned int expanded_width, unsigned int level, D3D11_USAGE usage)
```

Comment:
In my opinion, it's better to have src_pitch as argument, so feel free to move this "4 * expanded_width" into the callers.